cmd/podmanV2/root.go

```diff
@@ -1,61 +1,112 @@
 package main
 
 import (
+	"context"
 	"fmt"
 	"log/syslog"
 	"os"
 	"path"
+	"runtime/pprof"
 
 	"github.com/containers/libpod/cmd/podmanV2/registry"
 	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/containers/libpod/pkg/tracing"
 	"github.com/containers/libpod/version"
+	"github.com/opentracing/opentracing-go"
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	logrusSyslog "github.com/sirupsen/logrus/hooks/syslog"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
 var (
 	rootCmd = &cobra.Command{
-		Use:               path.Base(os.Args[0]),
-		Long:              "Manage pods, containers and images",
-		SilenceUsage:      true,
-		SilenceErrors:     true,
-		TraverseChildren:  true,
-		PersistentPreRunE: preRunE,
-		RunE:              registry.SubCommandExists,
-		Version:           version.Version,
+		Use:                path.Base(os.Args[0]),
+		Long:               "Manage pods, containers and images",
+		SilenceUsage:       true,
+		SilenceErrors:      true,
+		TraverseChildren:   true,
+		PersistentPreRunE:  preRunE,
+		RunE:               registry.SubCommandExists,
+		PersistentPostRunE: postRunE,
+		Version:            version.Version,
 	}
 
 	logLevels = entities.NewStringSet("debug", "info", "warn", "error", "fatal", "panic")
-	logLevel  = "error"
+	logLevel  = "warn"
```

Comment:
Why? There's a whole lot of warn-level logging we never want to display. I don't see why we'd want to change this convention at this late date.

Reply:
Fixed.